.evergreen/csfle/activate_venv.sh

```diff
@@ -2,14 +2,18 @@ if [ "Windows_NT" = "$OS" ]; then
   PYTHON_BINARY=C:/python/Python38/python.exe
 elif command -v /opt/python/3.6/bin/python3; then
   PYTHON_BINARY=/opt/python/3.6/bin/python3
-else
+elif command -v python3; then
   PYTHON_BINARY=python3
+elif command -v /opt/mongodbtoolchain/v2/bin/python3; then
+  PYTHON_BINARY=/opt/mongodbtoolchain/v2/bin/python3
+else
+  echo "error: unable to find a supported python3 executable"
 fi
 
 # create venv on first run
 if [ ! -d kmstlsvenv ]; then
    FIRST_RUN=1
-   virtualenv -p ${PYTHON_BINARY} kmstlsvenv
+   ${PYTHON_BINARY} -m virtualenv -p ${PYTHON_BINARY} kmstlsvenv
```

Comment:
Is this change needed? I'm concerned it may break some platforms.

Reply:
How could this break existing behavior given the remainder of the script assumes/requires a valid `${PYTHON_BINARY}`? Are there platforms that install `virtualenv` without a `python3` binary?

Reply:
I also am not sure how this would break existing behavior, but I do wonder if it's redundant with the following `-p {PYTHON_BINARY}`.

Reply:
In the past there were platforms where `virtualenv` exists but `python3 -m virtualenv` does not. I'm not 100% sure the problem still exists but if this change isn't required I think it's safer to remove it.

Reply:
@benjirewis It's not quite the same: `${PYTHON_BINARY} -m virtualenv` requests a `virtualenv` binary installed/associated with `${PYTHON_BINARY}`, whereas just `virtualenv` uses the system-provided binary, which may not be the same as that associated with `${PYTHON_BINARY}` (e.g. a Python2 binary, or a standalone binary installed via packages?). I suppose I was assuming too much to think that the Python binary and associated packages would be consistent on all platforms.

@ShaneHarvey If that is the case, I will undo this change, as I believe it wasn't strictly necessary to unblock related work.